src/main/java/com/amelie/guessthenumber/GuessTheNumber.java

```diff
@@ -1,9 +1,80 @@
 package com.amelie.guessthenumber;
 
+import java.net.URL;
+import java.util.Random;
+import java.util.ResourceBundle;
+import javafx.fxml.FXML;
+import javafx.fxml.Initializable;
+import javafx.scene.control.Button;
+import javafx.scene.control.TextArea;
+import javafx.scene.control.TextField;
+
 /**
  * Amelie's version of guess the number.
  */
 @SuppressWarnings("PMD.AtLeastOneConstructor")
-public class GuessTheNumber {
+public class GuessTheNumber implements Initializable {
+
+  @FXML
+  /* default */ Button btnSubmit;
+
+  @FXML
+  /* default */ TextField txtGuess;
+
+  @FXML
+  /* default */ TextArea txtOutput;
+
+  /* default */ int randomNum;
```

Comment:
This variable name is unclear what it represents. numToGuess might be better? Maybe something else?
```suggestion
  private int numToGuess;
```